api/db/models/user.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy import Column, Integer, String, Boolean
+from sqlalchemy.orm import relationship
+from ..base_class import Base
+from .supporter_petitions import supporter_petitions
+
+
+class User(Base):
+    __tablename__ = "user"
+
+    id = Column(Integer, primary_key=True, index=True)
+    firstname = Column(String, nullable=False)
+    lastname = Column(String, nullable=False)
+    email = Column(String, unique=True, index=True, nullable=False)
+    hashed_password = Column(String(128), nullable=False)
+    is_active = Column(Boolean, default=True)
+    is_superuser = Column(Boolean, default=False)
+    petitions = relationship("Petition",
```

Comment:
`supported_petitions` or  `signed petitions` would be more clear

Reply:
I fixed this